### ISSUE TITLE
💡[feat] list only unarchived experiments

### ISSUE THREAD

**Issue by ghstgs** [determined-ai/determined#6553]
### Describe the problem

`det e ls` lists all past experiments

### Describe the solution you'd like

could we get a flag to only list unarchived experiments with

det e ls

and all experiments when needed with

det e ls --include_archived

**Comment by rb-determined-ai** [comment]
Did you try `det e list --help`?

It shows:
```
  --all, -a        show all experiments (including archived and other users')
```

Does that meet your need?

**Comment by ghstgs** [comment]
thanks for the quick replies

maybe my feature request was not clear enough; what I would like is not to show more experiments but rather show less experiments; basically to not list archived experiments in `det e ls` 

`det e ls -a` would still show all as it presently does

**Comment by ioga** [comment]
currently `det e ls` hides individually archived experiments, and I agree it should also hide the ones from the archived products. we'll add this to our backlog.

thank you!

**Comment by ioga** [comment]
late update: this should've been fixed since 0.23.0.
